### DISCUSSION TITLE
Issues after newest update

### DISCUSSION THREAD

**Discussion by colivinggit** [livewire/livewire#7312] [7. Bugs (archived)]
### Livewire version

3.20

### Which PHP version are you using?

PHP 8.2

### Steps To Reproduce

On Coliving.com 
After upgrading to newest version, I get this issue:
[Error] TypeError: undefined is not an object (evaluating 'window.livewireScriptConfig["uri"]')
	(anonymous function) (livewire.js:491:113)
	Global Code (livewire.js:7589)
	

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I have provided code samples as text and NOT images.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by joshhanley** [comment]
@colivinggit Thanks for reporting. Can you please share your code that replicates the issue so we can investigate? Thanks!

**Reply by colivinggit** [reply]
All our pages produce this issue.
I'm now migrating from Mix to Vite, and it looks like this can fix the issues I experience.

**Reply by joshhanley** [reply]
@colivinggit You shouldn't have to do that if we've introduced a breaking change. Can you share your app.js and mix config so I can replicate it? Thanks

**Comment by ArtMin96** [comment]
Same issue here
**app.js**
```js
import './bootstrap';

document.addEventListener('alpine:init', () => {
    const theme = localStorage.getItem('theme') ?? 'system'

    window.Alpine.store(
        'theme',
        theme === 'dark' ||
        (theme === 'system' &&
            window.matchMedia('(prefers-color-scheme: dark)').matches)
            ? 'dark'
            : 'light',
    )

    window.addEventListener('theme-changed', (event) => {
        let theme = event.detail

        localStorage.setItem('theme', theme)

        if (theme === 'system') {
            theme = window.matchMedia('(prefers-color-scheme: dark)').matches
                ? 'dark'
                : 'light'
        }

        window.Alpine.store('theme', theme)
    })

    window
        .matchMedia('(prefers-color-scheme: dark)')
        .addEventListener('change', (event) => {
            if (localStorage.getItem('theme') === 'system') {
                window.Alpine.store('theme', event.matches ? 'dark' : 'light')
            }
        })

    window.Alpine.effect(() => {
        const theme = window.Alpine.store('theme')

        theme === 'dark'
            ? document.documentElement.classList.add('dark')
            : document.documentElement.classList.remove('dark')
    })
});
```

**vite.config.js**
```js
import { defineConfig } from 'vite';
import laravel, { refreshPaths } from 'laravel-vite-plugin';

export default defineConfig({
    plugins: [
        laravel({
            input: ['resources/css/app.css', 'resources/js/app.js'],
            refresh: [
                ...refreshPaths,
                'app/Livewire/**',
            ],
        }),
    ],
});
```

**Comment by ArtMin96** [comment]
Anything related to livewire does not work. I think the problem comes from this [PR](https://github.com/livewire/livewire/pull/7222)
![image](https://github.com/livewire/livewire/assets/29732308/3a47b978-3ae4-4301-b84e-724fb136ceb4)
![image](https://github.com/livewire/livewire/assets/29732308/54aced39-0069-4ee4-aae6-680e138b8acf)

**Reply by joshhanley** [reply]
@ArtMin96 Thanks for reporting! Any chance you could create a small repo that demonstrates the issue with as little code as possible? And I'll have a look at it in the morning to see if I can work out what is going on. Thanks!

**Reply by ArtMin96** [reply]
The problem arose because livewire assets were published. I published it because of some problems. But for now, there are no errors. Thanks for reply

**Reply by joshhanley** [reply]
@hackie-me I'd happily investigate it but I can't replicate the issue. I'm going to need someone to provide me with a small repo that demonstrates the issue so we can investigate.

If you can get that to me, I'll look at it with a matter of priority 🙂

Also you said you published your asset, how are you doing that?

Thanks!

**Reply by joshhanley** [reply]
@hackie-me Ah ok thanks for confirming 🙂 any chance you could share what the bug was here, incase others run into the same issue? Thanks!

**Comment by robsontenorio** [comment]
@joshhanley 

In my case issue is

![image](https://github.com/livewire/livewire/assets/118955/2c089885-1361-4555-b7c6-5c0a66ad5ef8)

(just used Safari IOS dev tools to check it)



The problem is it happens only on some devices, not others. And a hard refresh on browser solve it. 

But in case of Safari IOS, the refresh button does not solve and user stays "trapped" on that broken version. If user opens a anonymous tab works.


So, it is hard to have a reproduction steps here.
